### ISSUE TITLE
Silent mode still produces output

### ISSUE THREAD

**Issue by lwasyl** [arturbosch/detekt#1854]
## Expected Behavior
Silent mode produces no output if no issues are found

## Observed Behavior
Empty lines/newlines are printed out when `detekt` task is executed

## Steps to Reproduce
Relevant configuration piece:
```yaml
processors:
  active: false

console-reports:
  active: true
  exclude:
    - 'ProjectStatisticsReport'
    - 'ComplexityReport'
    - 'NotificationReport'
```

I do want findings to be printed out **if** there are violations. I'd expect with such configuration and no violations, `detekt` will be completely silent. Instead, it produces empty lines (if a task produces no lines, it's not visible it Gradle's output. Also there's clearly vertical space after the task, whereas other Gradle tasks don't have this space)

Actual output:
```
> Task :lint-checks:detekt


> Task :data:room-storage:detekt


> Task :common:testing:detekt


> Task :domain:detekt


> Task :app:detekt


> Task :data:graphql-networking:detekt


> Task :presentation:ui:detekt


```

## Your Environment
* Version of detekt used: 1.0.1
* Version of Gradle used (if applicable): 5.6.1
* Operating System and version: macos

**Comment by arturbosch** [comment]
Thanks for letting us know. Will try to find those newlines! :)
